### PR TITLE
move the configure_logging import to the app.py where it is used

### PR DESCRIPTION
**Pull request by martenson**
remove routes.middleware chatty logging as the PR is independent on this

### CODE REVIEW

File: lib/galaxy/webapps/tool_shed/app.py [hunk 1]
```diff
@@ -10,9 +10,11 @@
 from galaxy.openid.providers import OpenIDProviders
 from galaxy.util.dbkeys import GenomeBuilds
 from galaxy.web import security
+from galaxy.config import configure_logging
```
Comment (nsoranzo) [inline lib/galaxy/webapps/tool_shed/app.py]: @martenson Remember that we now sort the imports! This should go between `from galaxy import tools` and `from galaxy.managers.tags import CommunityTagManager`

Comment (martenson) [inline lib/galaxy/webapps/tool_shed/app.py]: yeah, I had a look at the sorting we are supposed to use and still do not understand it :D

Comment (nsoranzo) [inline lib/galaxy/webapps/tool_shed/app.py]: There are 3 principles:
- 3 sections, separated by an empty line:
  1) standard library imports;
  2) external libraries imports;
  3) internal and relative imports.
- Inside each sections, all `import lib` go before the `from lib import foo`
- Then sort alphabetically each subsection

Example: https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_smarkets.py

Comment (martenson) [inline lib/galaxy/webapps/tool_shed/app.py]: @nsoranzo I should be good now.
img_20160805_103241

Comment (nsoranzo) [inline lib/galaxy/webapps/tool_shed/app.py]: :joy:

**Pull request merged**